core/unix/os_exports.h

```diff
@@ -52,10 +52,8 @@
 
 #ifdef MACOS
```

Comment:
Be sure to watch the Mac Travis test on the merge to master (xref issues on Mac delays so we relegated to master-only, and issue on Travis not sending us emails on Mac failures...)

Reply:
(Alternatively temporarily enable Mac in .travis.yml for this PR)

Reply:
Ok.